yellowbrick/features/manifold.py

```diff
@@ -56,18 +52,13 @@
     "tsne": "t-SNE",
 }
 
-# Target type constants
-AUTO = "auto"
-SINGLE = "single"
-DISCRETE = "discrete"
-CONTINUOUS = "continuous"
-
 
 ##########################################################################
 ## Manifold Embeddings
 ##########################################################################
 
-class Manifold(FeatureVisualizer):
+
+class Manifold(ProjectionVisualizer):
     """
```

Comment:
Please review the docstring updates I made to `DataVisualizer`, would you mind applying those docstring changes to `ProjectionVisualizer` and now to `Manifold` as well? (e.g. primarily the changes to the parameter descriptions).

Reply:
I will have a look at it and make the necessary changes!